services/static-webserver/client/source/class/osparc/desktop/credits/Utils.js

```diff
@@ -20,12 +20,24 @@ qx.Class.define("osparc.desktop.credits.Utils", {
 
   statics: {
     DANGER_ZONE: 25, // one hour consumption
+    CREDITS_ICON: "@FontAwesome5Solid/database/",
```

Comment:
Not such a big fan of database icon in this context but just my 2c worth

Reply:
Just read the conversation above and what I commented during the daily